proto/src/lib.rs

```diff
@@ -0,0 +1,28 @@
+use std::{env, path::PathBuf};
+
+use tonic_build::FileDescriptorSet;
+
+const RPC_PROTO: &str = "rpc.proto";
+
+#[cfg(feature = "internal")]
+const STORE_PROTO: &str = "store.proto";
+
+#[cfg(feature = "internal")]
+const BLOCK_PRODUCER_PROTO: &str = "block_producer.proto";
+
+pub fn rpc_file_descriptor() -> Result<FileDescriptorSet, protox::Error> {
```

Comment:
Should we rename this to `rpc_api_descriptor()`? (and similar comment for the other two components).